source/isaaclab/test/assets/test_articulation.py

```diff
@@ -619,6 +619,7 @@ def test_out_of_range_default_joint_pos(self):
                         sim.reset()
                         # Check if articulation is initialized
                         self.assertFalse(articulation.is_initialized)
+                        self.assertRaises(ValueError, sim.reset)
```

Comment:
THe valueerror should check on L-619 when the sim.reset is called for the first time?